tests/rules/test_fix_file.py

```diff
@@ -14,47 +14,47 @@
 a.c:3:1: error: expected expression before '}' token
  }
   ^
-"""),
+"""),  # noqa
 
 ('clang a.c', 'a.c', 3, 1, '',
 """
 a.c:3:1: error: expected expression
 }
 ^
-"""),
+"""),  # noqa
```

Comment:
> I thought it would be easier to just use # noqa to ignore both the whitespace errors and the indent errors.

Yeah. At least until flake8 doesn't ignore just what it's told to. Maybe we're – or I am – getting a bit too meticulous here. These are test files, after all. Anyway, I guess that just appending `# noqa` to the one line where the `tests = (...)` tuple definition ends, is sufficient.